### ISSUE TITLE
Can't update saved project

### ISSUE THREAD

**Issue by futuresoup** [mozilla/popcorn-editor#67]
### Steps to reproduce:

1) Make some stuff then go to 'Project' and hit 'Save'
2) Make new changes to your project after saving
### Expected result:

You will be able to save your project again with the new updates registered
### Actual result:

The save button is greyed out and inactive, as if you had just saved. The save button does not recognize the fact that you've made new changes to the project that need to be updated.

**Comment by traceypooh** [comment]
yes, i think there's presently no way to save changes to the same project.

right now, changing to a different project title will allow a new item on archive.org to be made w/ the updated project, at least.

**Comment by futuresoup** [comment]
Perhaps the language of the button should change to read **Publish** instead of **Save** since that's more akin to what's being done. 'Save' implies that the project is still internal, which is not the case anyhow.

Either way, I imagine the user will have a strong expectation of being able to keep working on their project after they've published - so this may pose a significant challenge.

If we come up with interesting remix templates, perhaps we can socialize the use cases of Popcorn to be a one time sit down and publish experience rather than a persistent project that you continue to refine.

**Comment by Nolski** [comment]
Part of me wonders if the project tab should exist within the popcorn-editor repo at all. It'd be easy enough to open up javascript APIs where you can access the project data and then whichever site wants to use the editor can build their own UX around how to export/save/publish their project data.

Thoughts?
